vagrant/Vagrantfile

```diff
@@ -63,35 +31,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/home/vagrant/opt/monorail/"
         end
 
-        if ENV['TEST_GROUP']		
-          if ENV['TEST_GROUP'] =~  /esxi-5-5-min-install.v2.0.test/ || ENV['TEST_GROUP'] =~  /esxi-5-5-max-install.v2.0.test/		
-            config.vm.provision "file", source: "./dhcpd.conf", destination: "~/dhcpd.conf"		
-            config.vm.provision "shell" do |s|		
-            s.inline = "cp /home/vagrant/dhcpd.conf /etc/dhcp"		
-            s.privileged = true		
-            end		
-          end		
-        end
         if ENV['PYTHON_REPOS']
-          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
-          config.vm.provision "shell" do |s|		
-            s.inline = "apt-get install python-pip -y"		
-            s.privileged = true
-         end
           list = ENV['PYTHON_REPOS'].split(' ')
           list && list.each do |repo|
             config.vm.provision "shell" do |s|		
-            s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
+           s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
             s.privileged = true
             end
           end
         end
-        config.vm.provision "file", source: "./mongodb.conf", destination: "~/mongodb.conf"
-        config.vm.provision "shell" do |s|
-        s.inline = "cp /home/vagrant/mongodb.conf /etc"
```

Comment:
Per @changev ,  those conf files are all live inside RackHD/packer then build into  vagrant box 0.17
and will you remove the ```on-build-config/vagrant/*.conf```  as you no longer use them ?